README.adoc

```diff
@@ -6,6 +6,89 @@ image:https://codecov.io/gh/codeready-toolchain/toolchain-common/branch/master/g
 
 This repo is for controllers, libs, scripts, make files, etc to be shared between host and member operators.
 
+== Setting Up and Connecting Host and Member Clusters
+
+In a new terminal, execute the following commands
+```
+$ cd $GOPATH/github.com/codeready-toolchain/host-operator
+$ minishift start --profile host
+$ make up-local
+^C
+```
+
+```
+$ cd $GOPATH/github.com/codeready-toolchain/member-operator
+$ minishift start --profile member
+$ make up-local
+^C
+$ make add-member-to-host
+$ make add-host-to-member
+$ make up-local
+```
+
+Now that we are setup, we can add resources and watch the logs on the member-operator! To do so, open a new terminal and run the following commands
+```
+$ oc project toolchain-host-operator
+$ oc apply -f '/path/to/masteruserrecord.yaml'
+$ oc project toolchain-member-operator
+$ oc apply -f '/path/to/useraccount.yaml'
```

Comment:
This commands sequence won't work - don't forget that you use two minishift profiles ;-)

Reply:
in addition, in the previous code snippets, you started only the member operator, so you cannot access the host operator as it is not running

Reply:
@MatousJobanek I understand now :-) Done!! I tested out the steps in the order they are documented and it worked perfectly. As you suggested, it is more clear to list out the steps used instead of using make up-local. Especially better since we aren't starting the operator when it doesn't need to be started. Thanks for the advice and tips!